interface/modules/custom_modules/oe-module-dorn/public/results.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+/**
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once "../../../../globals.php";
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Common\Twig\TwigContainer;
+use OpenEMR\Modules\Dorn\ConnectorApi;
+use OpenEMR\Core\Header;
+//this is needed along with setupHeader() to get the pop up to appear
+
+$tab = "results";
+if (!AclMain::aclCheckCore('admin', 'users')) {
```

Comment:
acl correct?
(going to stop mentioning acl stuff on this review. Look like most of the acls needed to be better worked out (in terms of what acl to use and also a correct pageTitle for each one).